central/globaldb/v2backuprestore/formats/roxdbv1/file.go

```diff
@@ -0,0 +1 @@
+package roxdbv1
```

Comment:
I assume this is for the compiler when the tag filters the rest of the stuff out.  Might be worth a comment if so.

Reply:
correct, will add comment